fleet_management_api/api_impl/controllers/platform_hw.py

```diff
@@ -4,19 +4,17 @@
 import fleet_management_api.database.db_access as _db_access
 import fleet_management_api.database.db_models as _db_models
 import fleet_management_api.api_impl.obj_to_db as _obj_to_db
-from fleet_management_api.api_impl.api_logging import (
+from ...api_impl.api_logging import (
```

Comment:
Is there a reason to use two approaches to include modules (relative x absolute)?